_posts/2016-05-19-sample-job-req.md

```diff
@@ -0,0 +1,38 @@
+---
+title: A Template Job Posting for Open Source Office Lead
+author: bvanevery
+---
+
+I ran into several folks this past week at OSCON who expressed a keen interest in creating a dedicated role for Open Source at their respective companies. So what was stopping them? One simple thing: every single one of them was struggling to define exactly what that role means. Instinctively we all have a feeling of what an employee dedicated to Open Source might do, but when it comes time to write it down or try to convince payroll, it can be challenging. Below I have included a starting point for a job description of what a dedicated Open Source manager might do. If you are in this boat, I'd highly recommend that you also check out the slides from our [talk at OSCON](http://todogroup.org/blog/oscon-talk/) this year. In addition, the many blog posts we've [published](http://todogroup.org/blog) about why our respective companies run Open Source.
+
+# The Job Posting
+
+*Side note: if you use this template, try running it through analysis on https://textio.com/talent/ first.*
+
+## The Mission
+
+Our open source effort is currently lead by a multi-functional group of engineers and we are looking for a motivated, visionary individual to lead this effort and take Company Open Source to the next level.
+
+In this role, you'll work with our Engineering (Dev & Ops), Legal, Security, Business Ops, and Public Relations teams to help define what Open Source at Company means and build our open source community. Your day to day responsibilities will alternate between programming and several forms of program management. This is an exciting opportunity to work with all levels of the organization and leave a lasting impact here and on the engineering community at large.
```

Comment:
While I think it's important for the person heading the OSPO to have some technical expertise, I would guess the average day to day across members doesn't involve as much programming as the second sentence implies. I do think it's vital to call out "This isn't a typical program management role" though. How do people feel about

"Your day to day responsibilities will involve a mix of programming and technical skills, as well as several forms of program management."